File: keyphrase-extraction/ibm-watson-natural-language-understanding/IbmWatsonNaturalLanguageUnderstandingKeyphraseExtraction.java
NaturalLanguageUnderstanding service = new NaturalLanguageUnderstanding("2018-03-16", username, password);

KeywordsOptions options = new KeywordsOptions.Builder().build();

Features features = new Features.Builder()
    .keywords(options)
    .build();

AnalyzeOptions parameters = new AnalyzeOptions.Builder()
    .text(text)
    .features(features)
    .build();

AnalysisResults response = service
    .analyze(parameters)
    .execute();

List<KeywordsResult> keywords = response.getKeywords();

File: keyphrase-extraction/ibm-watson-natural-language-understanding/IbmWatsonNaturalLanguageUnderstandingKeyphraseExtraction.kt
// "text" is a string holding the input document.

val service = NaturalLanguageUnderstanding("2018-03-16", username, password)

val options = KeywordsOptions.Builder().build()

val features = Features.Builder()
    .keywords(options)
    .build()

val parameters = AnalyzeOptions.Builder()
    .text(text)
    .features(features)
    .build()

val response = service
    .analyze(parameters)
    .execute()

val keywords = response.keywords
